docs/index.md

```diff
@@ -0,0 +1,52 @@
+# cql2-rs
+
+**cql-rs** is command-line interface (CLI), Python package, and Rust crate for parsing, validating, and converting [Common Query Language (CQL2)](https://www.ogc.org/standard/cql2/).
+
+## CLI
+
+To install the CLI, you'll need [Rust](https://rustup.rs/).
+Then:
+
+```shell
+$ cargo install cql2-cli
```

Comment:
maybe it's just me, but I got confused because to me `cargo==rust` so in my head I though this section was about Rust.

The distinction between the Rust Crate and the CLI is maybe not necessary 🤷

Reply:
🤔 I do want to be able to point people to the CLI if they want to use it ... a follow-on PR https://github.com/developmentseed/cql2-rs/pull/38 adds the ability to install the CLI via PyPI _or_ cargo, does that confuse things even more?

> The distinction between the Rust Crate and the CLI is maybe not necessary 🤷

From a technical sense, it is — you can't `cargo install cql2` because the core library doesn't come with a CLI, the CLI is in a separate crate (aka package).

My hope is that eventually `cargo install thingy` will be as common as `pip install thingy` or `npm install thingy` but we're probably not there yet.